kubespawner/spawner.py

```diff
@@ -469,19 +487,6 @@ def get_pod_manifest(self):
         """
         Make a pod manifest that will spawn current user's notebook pod.
         """
-        # Add a hack to ensure that no service accounts are mounted in spawned pods
-        # This makes sure that we don't accidentally give access to the whole
-        # kubernetes API to the users in the spawned pods.
-        # See https://github.com/kubernetes/kubernetes/issues/16779#issuecomment-157460294
-        hack_volumes = [{
```

Comment:
Yes I agree with keeping `make_pod_spec` independent of the hack i just wanted to keep the `kubernetes_client` code in one place. I put it back now.